releasenotes/notes/root-cert-configmap-controller.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+releaseNotes:
+- |
+  **Improved** Istio feature TRUST_DISTRIBUTION="off" can be used to explicitly disable the creation of 'istio-ca-root-cert', if other means of trust distribution are used. If not set Istiod will continue to infer based on other settings.
```

Comment:
How can those means of trust mount the ca files into sidecar? And istiod can cooperate  set it to use correctly

Reply:
The CertManager one - TrustManager - creates exactly the same configmap that Istio is creating ( if configured), so no change in injector. 

For other methods - we need to adjust the injection template and config to allow user to specify projected volumes - but that would be a separate PR, this is only about Istiod not creating the configmap if user has other means of distribution

Reply:
Should this be per namespace? Perhaps a label on a namespace labeled for injection? 

I am hesitant to add support for this flag when we haven't decided on supporting the required alternatives (i.e. TrustManager)